ui-speedspacechart/src/components/layers/FrontInteractivityLayer.tsx

```diff
@@ -23,7 +23,7 @@ const FrontInteractivityLayer = ({
   return (
     <canvas
       id="front-interactivity-layer"
-      className="absolute ml-10 mt-2"
+      className="absolute ml-10"
```

Comment:
It seems like the ml-10 is erased by the `margin-left: 3rem;` in `#front-interactivity-layer`.

I don't see any difference if I remove `ml-10`

Reply:
done